mods/fossilslasharchaeology.json

```diff
@@ -0,0 +1,20 @@
+{
+    "format": 1,
+    "name": "Fossil / Archeology",
+    "authors": ["Team July"],
+    "desc": "Fossil / Archaeology adds many new relics, fossils, and living dinosaurs from ancient times.",
+    "versions": [
+        {
+            "name": "July 30, 2011",
```

Comment:
Ideally this should be `2011-07-30` so that it would be possible to sort by version numbers. It doesn't really matter since we don't actually do that, but that should probably be the preferred date format.

Don't worry about changing it. I'll do that when I merge this PR.